index.js

```diff
@@ -1497,6 +1497,26 @@
     return xBool !== yBool ? or(x, y) : xEmpty;
   });
 
+  //# both :: (a -> Boolean) -> (a -> Boolean) -> a -> Boolean
+  //.
+  //. Takes two unary functions that return a boolean and applies them
+  //. both to the supplied value, returning true iff both functions
+  //. evaluate to true.
+  //.
+  //. ```javascript
+  //. > S.both(S.test(/^a/), S.test(/.*f$/), 'abcdef')
+  //. true
+  //.
+  //. > S.both(S.test(/^a/), S.test(/.*e$/), 'abcdef')
+  //. false
+  //.
+  //. > S.both(S.test(/^b/), S.test(/.*e$/), 'abcdef')
+  //. false
+  //. ```
```

Comment:
As you intimated, `S.both(S.test(/^a/), S.test(/.*f$/))` can be written `S.test(/^a.*f$/)`. It would be nice to use predicates which cannot be combined in this way. How about these?

```javascript
> S.both(S.test(/a/), S.test(/b/), 'banana')
true

> S.both(S.test(/a/), S.test(/b/), 'orange')
false
```